uniprot-search/src/main/resources/gene-centric/search-fields.json

```diff
@@ -1,12 +1,12 @@
 [
   {
     "field": "accession_id",
-    "fieldValidRegex": "([O,P,Q][0-9][A-Z|0-9]{3}[0-9]|[A-N,R-Z]([0-9][A-Z][A-Z|0-9]{2}){1,2}[0-9])(-\\d+)*",
+    "fieldValidRegex": "([OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z]([0-9][A-Z][A-Z0-9]{2}){1,2}[0-9])(-[0-9]+)?",
```

Comment:
I think, we should file a separate ticket to write a bunch of positive and negative test cases to cover this regular expression. I don't feel comfortable with regular expression until it has been tested extensibly.

Reply:
i agree